### PR TITLE
Ports ambition system.

### PR DESCRIPTION
**Pull request by printer16**
Ports the ambition system from bay that lets antags set the ambition for themselves that displays at round end. Has no effect on the round. And removes a clown check.

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -195,6 +200,17 @@
 		if (isnull(new_memo)) return
 		memory = new_memo
 
+	else if (href_list["amb_edit"])
+		var/datum/mind/mind = locate(href_list["amb_edit"])
```
Comment (skull132) [inline code/datums/mind.dm]: This variable can be skipped and replaced with `src`. This is also unnecessary due to the same reason. `;amb_edit=\ref[src]`

**Comment by Arrow768** [comment]
It might be a good idea to prevent the ambitions from being changed after a certain amount of time and to pester them a bit at the round start to set them, if they have not.

**Comment by printer16** [comment]
I don't really want antags to feel compelled to follow a set goal as people won't use it as much. It's more to explain what their goal was to the entire server instead of making them feel forced to set a goal and follow it.

**Comment by printer16** [comment]
Changelog added and tested.

**Pull request merged**